### PR TITLE
v0.9.15 changelog update

### PR DESCRIPTION
**Pull request by Amazinite**
Updating the changelog with all the changes since v0.9.14 up to 7683e39.

### CODE REVIEW

**Comment by quyykk** [comment]
I think parallax can be considered big feature.

File: changelog [hunk 1]
```diff
@@ -1,3 +1,166 @@
...
+      * More Remnant bounty hunting jobs to hunt down the new Korath Dredgers. (@arkhne)
+      * Added an additional mission after the Eye opens in the Wanderer campaign to explain to the player that they need to wait for the next part of the campaign to start. (@MasterOfGrey, @Terin)
+      * Added "spacediving" jobs to human space that can have... unpredictable outcomes. (@petervdmeer)
+      * Three new missions in Coalition space exploring Coalition folklore. (@Arachi-Lover)
+      * Created a mission in Coalition space involving running into an unlikely character. (@Arachi-Lover)
+      * Created a hardpoint sprite for the nuclear missile. (@Amazinite)
+      * Added a new person ship. (@Brick63, @Zitchas)
+      * Created reverse thruster versions for human ion and plasma engines. (@ravenshining, @Zitchas)
+      * Created lighter versions of various human secondary weapons for easier use on smaller ships. (@Pointedstick, @ravenshining, @Zitchas)
+      * Added a pirate mission involving a mafia with a penitent for extorting unwilling victims. (@petervdmeer)
```
Comment (Wedge009) [inline changelog:38]: 'penitent' doesn't make sense to me in this context. Perhaps it was meant to be 'penchant'?

Comment (Amazinite) [reply changelog:38]: Yeah, that's a typo on my part, and a pretty egregious one at that.

Comment (Amazinite) [reply changelog:38]:
```suggestion
      * Added a pirate mission involving a mafia with a penchant for extorting unwilling victims. (@petervdmeer)
```

**Comment by Amazinite** [comment]
This PR was opened when the due date on the v0.9.15 milestone as October or November. Well we're well past that now and there's no new due date on the milestone, so I'm just going to merge what's done so far and we can update the rest when we do have a release date.

**Pull request merged**